src/StdUtils.sol

```diff
@@ -2,8 +2,10 @@
 pragma solidity >=0.6.2 <0.9.0;
 
 import "./console2.sol";
+import "./StdChains.sol";
 
-abstract contract StdUtils {
+abstract contract StdUtils is StdChains {
+    Vm private constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
```

Comment:
Should those functions be `StdCheats` instead?

Reply:
Sorry which functions? `StdChains` just exposes the `stdChains` state var which feels more like a util than a cheat, but I don't feel strongly

Reply:
Oh maybe you mean `assumeNoPrecompiles`? Hmm I guess maybe `StdCheats` is the better fit because it uses cheat codes? Maybe a clarification on the distinction between StdUtils and StdCheats would help me choose 😅

Reply:
What if we made `StdChains` a standalone Component, like all other, and import it in `Test` instead? Seems the use case is somewhat specific - for dealing with chains.

Reply:
I'm good with that! That seems better anyway, I forget why I made StdUtils inherit from it at this point anyway, but I don't see a reason why we need to

Reply:
It's because `stdChains` is initialized after our `_constructor` is run, so it overrides the setup.

Solution: move `stdChains` initialization into our `_constructor`, before setting the values.